core/src/impl/Kokkos_Combined_Reducer.hpp

```diff
@@ -203,18 +203,18 @@ KOKKOS_INLINE_FUNCTION auto _get_value_from_combined_reducer_ctor_arg(
 
 template <class T>
 KOKKOS_INLINE_FUNCTION auto _get_value_from_combined_reducer_ctor_arg(
-    T&& arg) noexcept ->
+    T&&) noexcept ->
     typename std::enable_if<is_view<typename std::decay<T>::type>::value,
                             typename std::decay<T>::type>::type::value_type {
-  return arg();
+  return typename std::decay<T>::type::value_type{};
```

Comment:
Plz resolve conflicts and prefer `std::decay_t` here and in the trailing return type

Reply:
Fixed.